src/qtism/cli/Render.php

```diff
@@ -351,7 +346,7 @@ private function runXhtml(XmlDocument $doc, XhtmlRenderingEngine $renderer)
             $header .= "${indent}<head>${nl}";
             $header .= "${indent}${indent}<meta charset=\"utf-8\">${nl}";
 
-            if (empty($title) !== false) {
+            if (empty($title)) {
```

Comment:
You may want to add the title when it's **not** empty.
```suggestion
if (!empty($title)) {
```

Reply:
You have a point. This makes sense. Indeed, this is inverting the existing logic (`!== false` is equivalent to `=== true`), which seemed to be faulty. Fortunately, this is some code used only to demo the possibilities in CLI, that is not used in production and that is not covered by tests. I'll changed that.
Thanks for this Misi! ;-)